### ISSUE TITLE
[BUG]: Excel file wasn't created

### ISSUE THREAD

**Issue by AntonShukanau** [IBMa/equal-access#1941]
### Project

accessibility-checker for Node

### Browser

Chrome

### Operating system

Windows

### Description
Here is my config which I'm using for creating html file with the results, but when I'm trying to replace outputFormat "html" in config to 'xlsx' - tool isn't creating anything
![image](https://github.com/IBMa/equal-access/assets/146948233/cff9f750-1e97-42c7-a82a-220bfd26626d)

Here is the code snippet for html creation and processing
![image](https://github.com/IBMa/equal-access/assets/146948233/7de51ca6-687a-4562-8b29-338e34868517)
Excel:
![image](https://github.com/IBMa/equal-access/assets/146948233/d351c871-0fd3-493f-8d48-f85fa76ee456)

### Steps to reproduce

All in the description block, sorry, can't say more :(

**Comment by AntonShukanau** [comment]
Found a discussion related to the same issue:
https://github.com/IBMa/equal-access/discussions/1836

**Comment by MHoov** [comment]
Convo #1836 was resolved by issues #1837

**Comment by AntonShukanau** [comment]
I'm passing Playwright page to getCompliance() method

**Comment by tombrunet** [comment]
@AntonShukanau To confirm, you're using the `accessibility-checker` page with Playwright? Since the Excel writes at the end of the session, there can be issues where it doesn't write if the regular Node shutdown process has any glitches. Try running aChecker.close() when your tests are complete.

**Comment by AntonShukanau** [comment]
Hi @tombrunet, yes, that's correct. 
![image](https://github.com/IBMa/equal-access/assets/146948233/5a2c91cc-9a88-4b48-a14d-f7b20812a301)
still didn't help
outputFormat: ["html", "xlsx"],
but it's creating an html file, if i left only "html" in config
![image](https://github.com/IBMa/equal-access/assets/146948233/617c3971-6ae3-4651-92e0-7934a94a0863)
it won't create anything :(

**Comment by shunguoy** [comment]
@AntonShukanau  can you please try to catch the error to see if anything happened?  also refer to [README](https://github.com/IBMa/equal-access/tree/master/accessibility-checker/src) for report generation?
```
try {
    .........
} catch (err) {
    console.error(err);
} finally {
    // close the engine
    await aChecker.close();
};
```
please let us know if this helps. If not, please also post your new code.  Thanks.

**Comment by AntonShukanau** [comment]
@shunguoy hi, all I've got:
![image](https://github.com/IBMa/equal-access/assets/146948233/64cfbf91-75db-4728-854a-6c3674305219)
![image](https://github.com/IBMa/equal-access/assets/146948233/b98ab319-4827-49cd-931d-89ee5697c2d6)

**Comment by shunguoy** [comment]
@AntonShukanau  it seems the Excel file path is wrong and the checker cannot find the file as the error message indicated: "No such file or directory, open ......".   Please correct the file path and run again.  If the issue is resolved, please close the issue. Thanx.

**Comment by AntonShukanau** [comment]
> @AntonShukanau it seems the Excel file path is wrong and the checker cannot find the file as the error message indicated: "No such file or directory, open ......". Please correct the file path and run again. If the issue is resolved, please close the issue. Thanx.

![image](https://github.com/IBMa/equal-access/assets/146948233/3a11187b-347e-4009-b8c5-8c3bedde7a59)
I don't think so since html here is creating

**Comment by tombrunet** [comment]
@AntonShukanau The different report types output in different ways. All of these are rooted in the outputFormat directory.

For "json", it will create one file per label, using the label as the start of the filename, and one file as a summary (`summary_(timestamp).json`) after the tests are complete. 
For "html", it will create one file per label, using the label as the start of the filename.
For "csv" and "xlsx", it will only generate a summary file, with a filename of `results_(timestamp).csv` or `results_(timestamp).xlsx` respectively after all of the testing is complete (not per test)

The summary, or results files will only generate if everything closes down cleanly and the expected after test functions run without errors. Sometimes, that doesn't happen and the `.close` needs to run (once, at the end of all of the tests). It looks like you're trying to generate an excel file per scan, but the excel is a summary of scans, so it's not designed to work that way.

**Comment by shunguoy** [comment]
Triage: close due to no response.
